scaffolds/csharp/README.md

```diff
@@ -0,0 +1,26 @@
+# C# Scaffold for Code Retreat
+
+## Install the .NET CLI and .NET Core
+
+There are several options. The easiest way is to work with a docker image ready
+with all the tools you need:
+
+```Bash
+docker build -t csharp-code-retreat .
+```
+
+The only downside is that you need to run the .NET CLI through Docker. 
+//To make it easy, we wrote a script `gradle.sh` wrapping the `docker` command and acting as the .NET CLI
```

Comment:
Yeah, I copy / pasted from the Java scaffold, but it's not done yet... I was able to run it in docker, but I don't know how to automate it (using a script, I guess).